src/lib/__tests__/Engineer.test.js

```diff
@@ -0,0 +1,44 @@
+// using Engineer constructor
+const Engineer = require("../../lib/Engineer");
+
+describe("Engineer", () => {
```

Comment:
See comment from Employee tests on moving the `instance` of the Engineer to the describe block, to avoid unnecessary repetition. Please address this in the other test files too.